carlos/rocker/css/style.css

```diff
@@ -0,0 +1,272 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
```

Comment:
Avoid using other value type that is not pixels for font-sizes. (Code principle: [KISS](https://medium.com/@devisha.singh/the-kiss-principle-in-software-development-everything-you-need-to-know-dd8ea6e46bcd) - Keep it super simple)